### ISSUE TITLE
Multi language issue

### ISSUE THREAD

**Issue by paolodina** [lewyuburi/nuxt-validate#14]
Related to #4, maybe I'm doing something wrong because I get this error:

```
  Error: Could not compile template
  /home/paolo/devel/projects/bvk/node_modules/nuxt-validate/plugin.js:
  validatorLocale is not defined
```

nuxt.conf.js:
```
    ['nuxt-validate', {
      lang: 'it',
      nuxti18n: true
    }],
    ['nuxt-i18n', {
      lazy: true,
      langDir: 'i18n/',
      strategy: 'prefix_except_default',
      defaultLocale: 'it',
      locales: [
        {
          code: 'it',
          iso: 'it',
          name: 'IT',
          file: 'it.js',
        },
        {
          code: 'en',
          iso: 'en',
          name: 'EN',
          file: 'en.js',
        }
      ]
    }]
  ],
```

Any help?

**Comment by aaharu** [comment]
thanks for report
it seems to my bug

**Comment by lewyuburi** [comment]
Fixed in v1.0.1 thanks both!
